src/validation.ts

```diff
@@ -229,6 +241,73 @@ export class OpenAPIValidator {
     return result;
   }
 
+  /**
+   * Validates a response against a prebuilt Ajv validator and returns the result
+   *
+   * @param {*} headers
+   * @param {(Operation | string)} [operation]
+   * @param {number} [statusCode]
+   * @param {string} [setMatchType] - one of 'any', 'superset', 'subset', 'exact'
+   * @returns {ValidationResult}
+   * @memberof OpenAPIRequestValidator
+   */
+  public validateResponseHeaders(headers: any, operation: Operation | string, statusCode: number, setMatchType?: string)
```

Comment:
Let's make statusCode optional.  (I'll do this also for validateHeaders, once it supports statusCodes)

We probably should default the statusCode like in mockResponseForOperation:
- first `2XX` response in `operation.response`
- `default`
- first response in `operation.responses`

Reply:
Sure :).